src/codegate/pipeline/extract_snippets/extract_snippets.py

```diff
@@ -118,6 +123,9 @@ def extract_snippets(message: str) -> List[CodeSnippet]:
                     if lang not in available_languages:
                         lang = None
 
+        #  just correct the typescript exception
+        if lang == "typescript":
```

Comment:
I wonder if it might be worth doing this quickly, as we may have to do more of these mappings:

```python
lang_map = {
    "typescript": "javascript"
}
lang = lang_map.get(lang, lang)
```